exercises/bob/src/Bob.re

```diff
@@ -0,0 +1,37 @@
+type speechType =
+  | Question
+  | Shouting
+  | Nothing
+  | Other;
+
+let shouting = (input: string) : bool => input === String.uppercase(input);
+
+let alphaChars = (input: string) : bool =>
+  [%bs.re "/[a-zA-Z-]/"] |> Js.Re.test(input);
+
+let question = (input: string) : bool =>
+  String.contains_from(input, String.length(input) - 1, '?');
+
+let analyseSpeech = (speech: string) : speechType => {
+  let trimmed = String.trim(speech);
+  if (String.length(trimmed) === 0) {
```

Comment:
Use ```==```. This is the value equality operator. ```===``` does reference equality which is less commonly useful.